keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -106,14 +107,66 @@ For the first implementation we do not want to support restore in the
 outside of Kubernetes. The restore is a container engine only operation
 in this first step.
 
-The forensic use case is targeted to be part of the next (1.24) release.
+The forensic use case is targeted to be part of the next (1.25) release.
```

Comment:
You could probably drop this line entirely so you don't have to keep updating it :)

Reply:
Good point. Removed it.